auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -245,11 +245,13 @@ import { appendAccountId } from 'sbc-common-components/src/util/common-util'
 export default class EntityManagement extends Mixins(AccountMixin, AccountChangeMixin, NextPageMixin) {
   @Prop({ default: '' }) readonly orgId: string
 
+  // for template
+  readonly CorpTypes = CorpTypes
```

Comment:
What is the point of this here? Is CorpTypes not exposed to be read after being imported?

Reply:
Good question. Answer: the template can't see this type unless it's (re)declared like this in the component's script.